apps/python-sdk/firecrawl/v2/types.py

```
@@ -775,12 +775,63 @@ class PaginationConfig(BaseModel):
     max_results: Optional[int] = Field(default=None, ge=0)
     max_wait_time: Optional[int] = Field(default=None, ge=0)    # seconds
 
+# Vector search types
+class VectorSearchFilters(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)